zerver/views/custom_profile_fields.py

```diff
@@ -60,18 +67,23 @@ def delete_realm_custom_profile_field(request: HttpRequest, user_profile: UserPr
 @require_realm_admin
 @has_request_variables
 def update_realm_custom_profile_field(request: HttpRequest, user_profile: UserProfile,
-                                      field_id: int, name: Text=REQ()) -> HttpResponse:
+                                      field_id: int, name: Text=REQ(),
+                                      field_data: str=REQ(default='')) -> HttpResponse:
     if not name.strip():
         return json_error(_("Name cannot be blank."))
 
+    error = validate_field_data(field_data)
+    if error:
+        return json_error(error)
```

Comment:
I think you should be able to pass this as a `validator` or `converter` into `REQ` and not need to do the explicit parsing here, right?